topics/server.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Valkey Server"
+linkTitle: "Valkey Server"
+weight: 1
+description: >
+    Manual for valkey-server, the Valkey server program
+---
+
+## Usage
+
+**`valkey-server`** [ _/path/to/valkey.conf_ ] [ _OPTIONS_ ] [**`-`**]\
+**`valkey-server`** **`-`**\
+**`valkey-server`** **`-v`** | **`--version`**\
+**`valkey-server`** **`-h`** | **`--help`**\
+**`valkey-server`** **`--test-memory`** _megabytes_\
+**`valkey-server`** **`--check-system`**
+
+## Description
+
+`valkey-server` is the Valkey database program. It's typically started by an init script, rather than manually.
```

Comment:
Do we want a normal description here. Seems weird to have a description of "It's typically started by an init script".

```suggestion
`valkey-server` is the Valkey database program. 
Valkey is an open source (BSD) high-performance key/value datastore that supports a variety workloads such as caching, message queues, and can act as a primary database.
Valkey can run as either a standalone daemon or in a cluster, with options for replication and high availability.

Valkey natively supports a rich collection of datatypes, including strings, numbers, hashes, lists, sets, sorted sets, bitmaps, hyperloglogs and more.
You can operate on data structures in-place with a expressive collection of commands.
Valkey also supports native extensibility with built-in scripting support for Lua and supports module plugins to create new commands, data types, and more.
```

Reply:
That text is in `topics/introduction.md`, man page **valkey-introduction**(7).

There's also `topics/index.md`, man page **valkey**(7), which is just a list of links to all the pages.

Until now, there wasn't even a man page for valkey-server. I don't know where it best belongs but I don't think we should have it duplicated in several places. We could use "See also".

Reply:
We should probably also link to `topics/license.md` in some places, because the 2nd clause says it should be done "in the documentation and/or other materials":

> Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.

Reply:
Linking to `valkey-introduction` seems pretty reasonable.

Reply:
I took some inspiration from https://salsa.debian.org/lamby/pkg-redis/-/blob/debian/sid/debian/redis-server.1 (Debian writes their own man pages when missing. I think it's a requirement for all programs in Debian.)